Formula/dieharder.rb

```diff
@@ -21,6 +16,8 @@ class Dieharder < Formula
     sha256 cellar: :any, sierra:         "8a40fb61aef5230ad77b3b851a6e8b6d575ff2adaa747c3b73a75cd203197945"
   end
 
+  deprecate! date: "2022-01-17", because: "there is an issue with the upstream's ssl website's cert"
```

Comment:
```suggestion
  # At the time of writing (2022-01-17), the webhome.phy.duke.edu server has
  # an incomplete SSL certificate chain, which causes an error on Linux
  # (`curl: (60) SSL certificate problem: unable to get local issuer
  # certificate`). We may be able to revert this deprecation if this issue is
  # fixed on the upstream server in the future.
  deprecate! date: "2022-01-17", because: "uses an upstream server with an incomplete SSL certificate chain"
```

`brew info` precedes the deprecation text with "Deprecated because it", so this suggestion also rewords the `because` string to read correctly in that context and potentially explain the situation more clearly.

Reply:
Done. Please note that this happens also with brewed curl on macOS.